packages/requisitions/src/RequestRequisition/api/hooks.ts

```diff
@@ -16,11 +16,10 @@ import {
   SortController,
   useSortBy,
   getDataSorter,
-  useHostContext,
   useNotification,
   useTableStore,
 } from '@openmsupply-client/common';
-import { RequestRequisitionQueries } from './api';
+import { getRequestQueries } from './api';
```

Comment:
Made the changes required by the above change in the `api` file, utilising `useRequestApi` more